src/main/java/uk/gov/hmcts/reform/cwrdapi/repository/UserTypeRepository.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.hmcts.reform.cwrdapi.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import uk.gov.hmcts.reform.cwrdapi.domain.UserType;
+
+@Repository
+public interface UserTypeRepository extends JpaRepository<UserType,Long> {
```

Comment:
we can use simpleRepository if we have not any method for get. SimpleRepository.save() we can use

Reply:
same as the above.